docs/cep/query-guide/functions/geospatial/stationary.md

```diff
@@ -25,8 +25,21 @@ This function will return `true` when the object (defined in terms of longitude
 ## Example 1
 
 ```sql
-@info(name = 'query1')
-geo:stationary(km-4354,0,0, 110574.61087757687)
+geo:stationary('km-4354', 0, 0, 110574.61087757687)
 ```
 
-This example returns `true` because the given geofence with ID `km-4354` is considered stationary within the given radius of `110574.61087757687` from the reference point `(0, 0)`.
+In this example, the `geo:stationary()` function checks if the given geofence with ID `km-4354` is considered stationary within the given radius of `110574.61087757687` from the reference point `(0, 0)`. The function returns `true` if the geofence is stationary.
+
+## Example 2
+
+```sql
```

Comment:
And this

Reply:
@StoyanTc / @diana-macrometa -  Here also we have the same problem as for geo:proximity,

It seems the last parameter radius double for geo:stationary stream processor is not accepting dynamic values... The SW only validates if we give a hardcoded double value to the radius parameter..
Is this expected or a bug??

2. @StoyanTc Even if I hardcode radius to say 0.5, the SW validates but it doesn't work, I don't receive anything in OutputGeoStream, can you check if there is some bug or if the example I used is wrong?? Below are the details...

 Input given to InputGeoStream is `{"geofenceId":"person234", "longitude": 9.876657, "latitude": 80.897648}`

SW used is:
`CREATE STREAM InputGeoStream (geofenceId string, longitude double, latitude double);
CREATE STREAM OutputGeoStream (id string, stationaryStatus bool);

@info(name = 'StationaryStatusCheck')
INSERT INTO OutputGeoStream
SELECT geofenceId AS id, stationary AS stationaryStatus
FROM InputGeoStream#geo:stationary(geofenceId, longitude, latitude, 0.5);`

Reply:
@koshy-thomas - There is an issue into your SW. You have not defined `SINK` stream and `OutputGeoStream` is by default `Source`.

Reply:
I made the correction, the stream worker is still invalid.

Reply:
@diana-macrometa - Yes, I know, it's `CEP-586` created for.

Reply:
So, what should we do with this PR?

Reply:
If it will work when the bug is fixed, then we can just ignore this for now, or we can hide the topic until the bug is fixed.

Reply:
@diana-macrometa - Let's park it for now. This will be a bug fix which will require doc's changes.

Reply:
By "park it," do you mean hide the topic? I mean, the topic exists now in Prod.

Reply:
@diana-macrometa - I'm not sure what's the best solution in such case. The base sample would work but the stream processors like geo:proximity() and geo:stationary() have issues.

Reply:
I recommend that we temporarily hide the ones we do not want people to use. I can unhide them after they are fixed.

Reply:
Agreed!

Reply:
I hid geo:proximity and geo:stationary. Do any others need to be hidden?